### PR TITLE
Find maximum binary tree

### PR DESCRIPTION
**Pull request by TrunkOfUkuleles**
[ x ] Top-level README “Table of Contents” is updated
[ x ] Feature tasks for this challenge are completed
[ x ] Unit tests written and passing
[ x ] “Happy Path” - Expected outcome
[ x ] Expected failure
[ x ] Edge Case (if applicable/obvious)
[ x ] README for this challenge is complete
[ x ] Summary, Description, Approach & Efficiency, Solution
[ x ] Link to code
[ x ] Picture of whiteboard

### CODE REVIEW

File: challenges/maxBinaryTree/__tests__/maxTree.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+
+const Node = require('../node.js')
+const { BinaryTree , BinarySearchTree } = require('../tree.js');
+
+describe("Stacks and Queues Testing", () => {
```
Comment (lizkavalski) [inline challenges/maxBinaryTree/__tests__/maxTree.test.js:7]: Still need to change to Trees
